jobsdb/jobsdb.go

```diff
@@ -1213,26 +1213,29 @@ func (jd *Handle) getTableSize(jobTable string) int64 {
 func (jd *Handle) checkIfFullDSInTx(tx *Tx, ds dataSetT) (bool, error) {
 	if jd.conf.maxDSRetentionPeriod.Load() > 0 {
 		var minJobCreatedAt sql.NullTime
-		sqlStatement := fmt.Sprintf(`SELECT MIN(created_at) FROM %q`, ds.JobTable)
+		sqlStatement := fmt.Sprintf(`SELECT created_at FROM %q ORDER BY job_id ASC LIMIT 1`, ds.JobTable)
```

Comment:
Question: Why `SELECT MIN(created_at)` is more performant than `SELECT created_at ORDER BY job_id ASC LIMIT 1`?

Reply:
Generally speaking, MIN/MAX is more performant than Order By Limit. https://medium.com/@hamida.meknassi/min-max-vs-order-by-limit-performance-dfb6420701ee

Reply:
Can we mention this in PR description as well so that it can provide more context on why the change was made?

Reply:
yeah 
good idea